plugins/MauticCloudStorageBundle/Form/Type/RackspaceType.php

```diff
@@ -24,7 +25,7 @@ class RackspaceType extends AbstractType
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->add('serverLocation', 'choice', [
+        $builder->add('serverLocation', ChoiceType::class, [
             'label'   => 'mautic.integration.Rackspace.server.location',
             'choices' => [
```

Comment:
Symfony 2.7 introduced a BC break that used option values as array keys and option labels as the array values. But in Symfony 3.4, you have to use the label as the key. But in order to test it in Symfony 2.8, you have to add the choices_as_values => true option to revert the BC behavior introduced in Symfony 2.7. It's confusing I know. See the engineering notes in https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit#

But bottom line is any choice lists in Mautic 3 need to be like this:

```
'choices' => [
    'mautic.integration.Rackspace.server.location.us' => 'us',
    'mautic.integration.Rackspace.server.location.uk' => 'uk',
],
'choices_as_values' => true,
```

Reply:
@cesarZubillaga Just have to switch around the label and key in the choices array and then this will be good.